.ci/unit_tests/Base_Engine_Tests/Base_Engine_Tests.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Missing postbuild to move this DLL to the assemblies folder.

`if not "$(ConfigurationName)" == "Test" (xcopy "$(TargetDir)$(TargetFileName)"  "C:\ProgramData\BHoM\Assemblies" /Y)`